src/components/seo/meta.tsx

```diff
@@ -14,13 +14,13 @@ const Meta: React.SFC<Props> = ({
   defaultTitle,
   title,
   description,
-  separator = ' - ',
+  separator = '',
   noindex = false,
   children,
 }) => {
   return (
     <Helmet
-      titleTemplate={`%s${separator}Web Design | App Development | Newcastle`}
+      titleTemplate={`%s${separator}`}
```

Comment:
If a separator like '-' is passed in to the meta component the page title will end up being 'title-'. It would be better to have the titleTemplate just be titleTemplate={`%s`} and remove the separator from the props and interface entirely.

Reply:
Removed as requested!